src/switchcraft_advanced/services/intune_service.py

```
@@ -0,0 +1,362 @@
+
+        payload = {
+            "target": {
+                "@odata.type": "#microsoft.graph.groupAssignmentTarget",
+                "groupId": group_id
+            },
+            "intent": intent,
+            "settings": None
+        }
+
+        try:
+            resp = requests.post(url, headers=headers, json=payload)
```

Ruff (0.14.8)

Probable use of `requests` call without timeout

(S113)